addons/account_peppol/models/account_move.py

```diff
@@ -21,6 +21,19 @@ class AccountMove(models.Model):
         copy=False,
         readonly=True,
     )
+    ref = fields.Char(compute='_compute_payment_reference', readonly=False, store=True)
```

Comment:
I need to use and extend the `_compute_payment_reference` method, since it has no `@api.depends` in the original `account.move` model (on purpose) and another compute method would not get triggered then when doing a `@api.depends('payment_reference')`.